pkg/js-search/.gitignore

```diff
@@ -0,0 +1,8 @@
+.DS_Store
+/.bower
+/.idea
+/bower_components
+/node_modules
+/npm-debug.log
+/dist
+junit.xml
```

Comment:
I would say that only `junit.xml` is necessary here, as `node_modules`, `npm-debug.log`, `.DS_Store` and `dist` are matched by root `.gitignore`'s `**/dist`, (...etc) and I don't think we're using bower, bower components, idk what `.idea` is for.